libs/core/src/components/pds-toast/pds-toast.scss

```diff
@@ -0,0 +1,176 @@
+:host {
+  --animation-duration: 0.3s;
+  --animation-timing: cubic-bezier(0.4, 0, 0.2, 1);
+  --padding-inline: var(--pine-dimension-lg);
+  --padding-inline-desktop: var(--pine-dimension-2xl);
+  --sizing-height-default: 68px;
+  --sizing-min-width: calc(var(--sizing-total-width) - (var(--padding-inline) * 2));
+  --sizing-min-width-desktop: calc(var(--sizing-total-width) - (var(--padding-inline-desktop) * 2));
+  --sizing-total-width: 350px;
+
+  box-sizing: border-box;
+  display: block;
+  font: var(--pine-typography-body-medium);
+}
+
+.pds-toast {
+  align-items: center;
+  animation: pds-toast--animate-in var(--animation-duration) var(--animation-timing);
+  background-color: var(--pine-color-primary);
+  border-radius: var(--pine-dimension-xs);
+  box-shadow: var(--pine-box-shadow);
+  color: var(--pine-color-text-primary);
+  display: flex;
+  height: var(--sizing-height-default);
+  justify-content: flex-start;
+  margin-block-end: var(--pine-dimension-2xs);
+  max-width: 90vw;
+  min-width: var(--sizing-min-width);
+  padding: 0 var(--padding-inline);
+  position: relative;
+  width: fit-content;
+
+  @media screen and (max-width: 767px) {
+    height: 36px;
+    max-width: calc(100vw - (var(--pine-dimension-2xs) * 2));
+    min-width: auto;
+    padding-inline: var(--pine-dimension-2xs);
+  }
+
+  &--danger {
+    background-color: var(--pine-color-danger);
+  }
+
+  &--loading {
+    .pds-toast__loader {
+      align-items: center;
+      display: flex;
+      justify-content: center;
+      margin-inline-end: var(--pine-dimension-sm);
+      position: relative;
+
+      &-spinner {
+        animation: pds-toast--rotate 2s linear infinite;
+        height: 20px;
+        width: 20px;
+      }
+
+      &-path {
+        animation: pds-toast--dash 1.5s ease-in-out infinite;
+        stroke-dasharray: 1, 200;
+        stroke-dashoffset: 0;
+        stroke-linecap: round;
+      }
+    }
+  }
+}
+
+.pds-toast__icon {
+  color: var(--pine-color-text-primary);
+  display: inline-flex;
+  margin-inline-end: var(--pine-dimension-sm);
+}
+
+.pds-toast__message {
+  font: var(--pine-typography-body-sm-medium);
```

Comment:
The Figma file lists the text as regular font size and font weight. Can we update this to match?